### ISSUE TITLE
Save password to keychain

### ISSUE THREAD

**Issue by unixb0y** [pstadler/keybase-gpg-github#32]
Hello everybody,
Thanks for the instructions but the Pinentry-Mac program doesn't save the password even when I tick the box. You describe to do something with gpg-agent-info but I don't have that file.
Is there something new (something changed) since the writing of the article?

Thanks in advance!
gpg (GnuPG) 2.2.11

**Comment by pstadler** [comment]
I think this can be closed with #33. Please reopen if this is still a problem.

**Comment by unixb0y** [comment]
Nope, didn't work. Pinentry opens (the GUI version) but even if I click "Save in keychain", it just doesn't.
